controllers/src/test/scala/cromwell/pipeline/controller/ProjectConfigurationControllerTest.scala

```diff
@@ -21,50 +20,63 @@ class ProjectConfigurationControllerTest extends AsyncWordSpec with Matchers wit
 
   "ProjectConfigurationController" when {
     val accessToken = AccessTokenContent(TestUserUtils.getDummyUserId)
+    val projectId = TestProjectUtils.getDummyProjectId
     val configuration = ProjectConfiguration(
       ProjectConfigurationId.randomId,
-      TestProjectUtils.getDummyProjectId,
+      projectId,
       active = true,
       List(
         ProjectFileConfiguration(Paths.get("/home/file"), List(FileParameter("nodeName", StringTyped(Some("hello")))))
       ),
       ProjectConfigurationVersion.defaultVersion
     )
+    val configurationAdditionRequest = ProjectConfigurationAdditionRequest(
+      id = configuration.id,
+      active = configuration.active,
+      projectFileConfigurations = configuration.projectFileConfigurations,
+      version = configuration.version
+    )
 
     "update configuration" should {
       val error = new RuntimeException("Something went wrong")
 
       "return success for update configuration" in {
-        when(configurationService.addConfiguration(configuration, accessToken.userId)).thenReturn(Future.unit)
-        Put("/configurations", configuration) ~> configurationController.route(accessToken) ~> check {
+        when(configurationService.addConfiguration(projectId, configurationAdditionRequest, accessToken.userId))
+          .thenReturn(Future.unit)
+        Put(s"/projects/${projectId.value}/configurations", configurationAdditionRequest) ~> configurationController
```

Comment:
```suggestion
        Put(s"/projects/$projectId/configurations", configurationAdditionRequest) ~> configurationController
```
Same for other places

Reply:
If we change this, then we get somthing like this "/project/Project("{PROJECT_UUID}")/configuration"

Reply:
Oh. Ok, nevermind then